experimental/packages/opentelemetry-instrumentation-xml-http-request/test/unmocked.test.ts

```diff
@@ -46,7 +45,11 @@ describe('unmocked xhr', () => {
       spanProcessors: [testSpans],
     });
     registerInstrumentations({
-      instrumentations: [new XMLHttpRequestInstrumentation()],
+      instrumentations: [
+        new XMLHttpRequestInstrumentation({
+          semconvStabilityOptIn: 'http',
```

Comment:
nit: is this optin config necessary in this test?

Reply:
No, not really. 

It *was* necessary to make the old version of this test (see changes below) *fail*.  Before this PR, this test was using the presence of the `SEMATTRS_HTTP_RESPONSE_CONTENT_LENGTH` span attr as a sign that `PerformanceTiming` resource usage was working (implementation in `addSpanNetworkEvents` from sdk-trace-web).  When using only the newer semconv (via `semconvStabilityOptIn: 'http',`)  that attr is not added -- for a different reason.

In a perfect "lots of tests have no maintenance or cognitive overhead"-world, this test would be run for all three OLD, STABLE, and DUPLICATE cases ... but I felt that was overkill given the exhaustive tests in xhr.test.ts.